src/ui/server/constants/field-ids/insurance/your-buyer/index.ts

```diff
@@ -0,0 +1,10 @@
+export const yourBuyerPageVariables = () => ({
+  FIELDS: {
+    BUYER_COUNTRY: {
+      ID: 'countryId',
+      NAME: 'countryName',
+      LABEL: 'Country',
+      ERROR_MESSAGE: 'Select your buyer country',
+    },
+  },
+});
```

Comment:
Field IDs should be purely field IDs... Page variables should be in the controller, but pulling things in from constants/content strings.

- Labels should come from field content strings in: `content-strings/fields/insurance/your-buyer`
- Error message should come from content strings: `content-strings/error-messages`
- `NAME` shouldn't be required